pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -674,6 +674,10 @@ form {
   display: flex;
   align-content: baseline;
 
+  &.white {
+    background: var(--neutral_white);
+  }
```

Comment:
Added this class to use on `.action-block` elements that need a white background.